scrapers/EastBoys.yml

```diff
@@ -0,0 +1,39 @@
+name: "EastBoys"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - eastboys.com/en/tour/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="row cover_detail mt-3 mt-sm-5"]/h2[@class="latest"]/text()
+      Date:
+        selector: //div[@class="update_date text-light"]/text()
+        postProcess:
+          - replace:
+              - regex: '(Added:\s)(\d\d\.\d\d\.\d{4})'
+                with: $2
+              - regex: '(\d\d)(\.)(\d\d)(\.)(\d{4})'
+                with: "$1 $3 $5"
+              - regex: '(\d\d)\s(\d\d)\s(\d{4})'
+                with: "$2 $1, $3"
+          - parseDate: 01 02, 2006

```

Comment:
Seems a lot of changing, this seems to work ??
```Date:
        selector: //div[@class="update_date text-light"]/text()
        postProcess:
         - parseDate: 02.01.2006

Reply:
That does seem to work perfectly, so I went ahead and fixed that. This is my first time making a scraper (or anything similar), so thanks again!

Reply:
No worries I am still learning everyday myself. Thanks for making a contribution as it helps everybody out.